src/main/java/uk/gov/hmcts/reform/roleassignment/auditlog/LoggerAuditRepository.java

```diff
@@ -19,7 +19,10 @@ public LoggerAuditRepository(AuditLogFormatter auditLogFormatter) {
 
     @Override
     public void save(final AuditEntry auditEntry) {
-        String msg = logFormatter.format(auditEntry);
-        LOG.info(msg);
+        try {
+            LOG.info("Message {}", logFormatter.format(auditEntry));
+        }catch (Exception e){
```

Comment:
@aashishaggarwal123 this has to check. fortify failing because exposing sensitive data into logs